content/applications/finance/accounting/bank/setup/foreign_currency.rst

```diff
@@ -2,116 +2,116 @@
+Configuration --> Settings --> Currencies` and make sure the :guilabel:`Allow Multi-currencies` box
+is ticked. Provide a **Currency Exchange Gain / Loss** account and :guilabel:`Save`.
 
 Configure currencies
 --------------------
 
-Once the Odoo is configured to support multiple currencies, you should
-activate the currencies you plan to work with. To do that, go to the menu
-:menuselection:`Configuration --> Currencies`. All the currencies are created by default,
-but you should activate the ones you plan to support (to activate a
-currency, check its "Active" field).
-
-After having activated the currencies, you can configure the parameters
-to automate the currency rate update. These options are also in the
-settings of the Accounting application, in the bottom of the page:
+Once Odoo is configured to support multiple currencies, activate the currencies you plan to work
+with. To do that, go into :menuselection:`Accounting --> Configuration --> Currencies` or in
+:guilabel:`Activate Other Currencies` that appears under :guilabel:`Multi-Currencies` once the box
+has been checked. All currencies are created by default, but only a few are active from the start.
 
-.. image:: foreign_currency/foreign01.png
-   :align: center
+Once the currencies are activated, you can choose to **automate** the currency rate update, or leave
+it on **manual**. To configure the rate update, under the same :guilabel:`Currencies` menu, check
+:guilabel:`Automatic Currency Rates` and set :guilabel:`Interval` to your desired frequency. You
+also have the option to choose the :guilabel:`Service` you wish to obtain currency rates from.
 
-Click on the **Update Now** link to update the currency rates now.
+Click on the :guilabel:`Update Now` button to update the currency rates manually.
 
 Create a new bank account
 -------------------------
 
-In the accounting application, we first go to :menuselection:`Configuration -->
-Accounting / Bank account`, and we create a new one.
+In the accounting application, go to :menuselection:`Accounting --> Configuration --> Journals` and
+create a new one. Fill in the different fields, and :guilabel:`Save`.
 
-.. image:: foreign_currency/foreign02.png
+.. image:: foreign_currency/foreign-journal.png
    :align: center
+   :alt: Example of created bank journal
 
-Once you save this bank account, Odoo will create all the documents for
-you:
+Upon creation of the bank account, Odoo automatically creates an **account**, which is linked to the
+created journal and which can be found under :menuselection:`Accounting --> Configuration --> Chart
+of Accounts`.
 
-- An account in the trial balance
+Vendor bill in a foreign currency
+=================================
 
-- A journal in your dashboard
+To pay a bill in a foreign currency, simply select the currency type in the :guilabel:`Currency`
+field and register the payment. Odoo will automatically post the foreign **exchange gain or loss**
```

Comment:
```suggestion
To pay a bill in a foreign currency, simply select the currency next to the :guilabel:`Journal`
field and register the payment. Odoo automatically posts the foreign **exchange gain or loss**
```
- Since there is no field named 'Currency', no guilabel. Which means we should also be more precise about it's location (although it's visible in the screenshot)
- Talking about screenshots, we usually avoid highlighting elements with red rectangles nowadays. I guess here it could be warranted since the currency field isn't named, but I would rather be more precise in your text rather than go for "Look at that screenshot!" approach.
- Shouldn't you first select a journal? The currency linked to that journal is automatically selected. Of course you change the currency if needed. I might be missing something here since I'm an accounting noob 🤷‍♂️ 
- No future 🎸 unless necessary (doesn't feel that way here)

Reply:
The 'vendor bill' is selected by default as it's used in most cases, the other one would be 'Expenses'. Neither fall within the journal type 'Bank' because they are not related to currencies, so the user does not need to select a journal before the currency. Not for currencies, at least.